src/v2/guide/components-slots.md

```diff
@@ -1,10 +1,12 @@
 ---
-title: Slots
-type: guide
-order: 104
+título: Slots
+tipo: guía
+orden: 104
 ---
 
-> Esta página asume que usted ya ha leído [Conceptos Básicos de Componentes](components.html). Lea eso primero si usted es nuevo con respecto a componentes.
+> Esta página asume que ya ha leído [Conceptos Básicos de componentes](components.html). Léalo primero si es nuevo en los componentes.
```

Comment:
Let's keep the capital `C` in `Componentes`. I also prefer the following mix of the previous wording and yours:
```suggestion
> Esta página asume que ya ha leído [Conceptos Básicos de Componentes](components.html). Léalo primero si es nuevo con respecto a componentes.
```

Thoughts on this @sxcamacho ?

Reply:
Hi guys, I agree with you @Tintef . `Léalo primero si es nuevo con respecto a componentes` sounds better.